src/Hacks/thirdperson.cpp

```diff
@@ -5,6 +5,7 @@
 #include "../interfaces.h"
 
 QAngle fakeang= {0.0f, 0.0f, 0.0f}, realang = {0.0f, 0.0f, 0.0f};
+static bool buttonToggle = false;
```

Comment:
Global variable already has static duration. no need to explicitly write it here.